binary-string-types.ipynb

```diff
@@ -0,0 +1,530 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variable sized types\n",
+    "\n",
+    "## Variable length binary, large binary, string and large string\n",
+    "\n",
+    "Binary and string types share the same physical layout with where the string type is utf-8 binary and will produce an invalid result if the bytes are not valid utf-8.\n",
+    "\n",
+    "The difference the between binary/string and large binary/string is in the offset type. In the first case that is `int32` and in the second it is `int64`. Second in generally used for data larger than 2GB.\n",
```

Comment:
```suggestion
    "The difference between binary/string and large binary/string is in the offset type. In the first case that is `int32` and in the second it is `int64`. Second in generally used for data larger than 2GB.\n",
```

Reply:
About "Second in generally used for data larger than 2GB". I would maybe turn it a around and explain this as a limitation of the types using 32 bit offsets that they have a max size of 2GB for one array. 
(because in practice people still use the non-large variants for bigger data, but then you need multiple chunks)